client/lib/user/utils.js

```diff
@@ -8,8 +8,19 @@ var debug = require( 'debug' )( 'calypso:user:utilities' ),
  * Internal dependencies
  */
 var user = require( 'lib/user' )();
+var addQueryArgs = require( 'lib/url' ).addQueryArgs;
```

Comment:
is this file not transformed through Babel? if it _is_, let's use `import { addQueryArgs } from 'lib/url'` here instead